### ISSUE TITLE
delete in the lines "head__".

### ISSUE THREAD

**Issue by fandongminghitmt** [pgii/ARKitGenesis8#1]
After opening the file with a text editor and delete in the lines "head__".

I didn't find head_

what's this means ?

**Comment by pgii** [comment]
### When exporting, select the ARKitG8F_Export_Rules.csv file in Morph Export Rules
You did it?

**Comment by fandongminghitmt** [comment]
![微信截图_20201021171540](https://user-images.githubusercontent.com/39093961/96699684-224cb380-13c1-11eb-9b0d-38ab0f5d2f1f.png)

**Comment by fandongminghitmt** [comment]
as you see

**Comment by fandongminghitmt** [comment]
delete it ?

**Comment by pgii** [comment]
You do not need to delete.
Everything should be fine.
Try to remove all unnecessary objects from the scene.

**Comment by fandongminghitmt** [comment]
![微信截图_20201021172514](https://user-images.githubusercontent.com/39093961/96700787-65f3ed00-13c2-11eb-94cf-c476d6a6e364.png)

**Comment by fandongminghitmt** [comment]
Sorry , I didn't see the morph.

**Comment by fandongminghitmt** [comment]
When I import the model, I turn on the (impot the morph)

**Comment by pgii** [comment]
This means your morphs are not exported from Daz3d.
Try the same operations with the base model.
Do you see ARKit options in Daz?

**Comment by fandongminghitmt** [comment]
no, I didn't see ARKIT options in DAZ

**Comment by fandongminghitmt** [comment]
![微信截图_20201021173736](https://user-images.githubusercontent.com/39093961/96702218-20d0ba80-13c4-11eb-98f0-676b9457e4bd.png)

**Comment by fandongminghitmt** [comment]
It looks like this

**Comment by fandongminghitmt** [comment]
it's facemojo

**Comment by pgii** [comment]
Then you need to do this.

**Copy the contents of the "ARKitG8F" folder to the "My DAZ 3D Library" folder.**

![ARKitGenesis8Screenshot02](https://user-images.githubusercontent.com/4399076/96702413-279f0380-13ab-11eb-8a61-862a14ccd26d.png)

**Comment by fandongminghitmt** [comment]
![微信截图_20201021173948](https://user-images.githubusercontent.com/39093961/96702548-6db49100-13c4-11eb-9916-d3f6416ae4d0.png)

**Comment by fandongminghitmt** [comment]
I did it, like this, then export.

**Comment by fandongminghitmt** [comment]
but no work.

**Comment by fandongminghitmt** [comment]
C:\Daz 3D\Applications\Data\DAZ 3D\My DAZ 3D Library

**Comment by pgii** [comment]
You need to copy the contents of the folder and not the folder itself.

**Comment by fandongminghitmt** [comment]
I see, let try again

**Comment by fandongminghitmt** [comment]
![微信截图_20201021174332](https://user-images.githubusercontent.com/39093961/96703027-f3d0d780-13c4-11eb-892f-6ccc4371c575.png)

**Comment by fandongminghitmt** [comment]
sorry, I make a mistake, let me try again.

**Comment by fandongminghitmt** [comment]
![微信截图_20201021174852](https://user-images.githubusercontent.com/39093961/96703660-b3258e00-13c5-11eb-9aa4-44d94f142039.png)

**Comment by fandongminghitmt** [comment]
![微信截图_20201021175028](https://user-images.githubusercontent.com/39093961/96703858-ed8f2b00-13c5-11eb-9d51-9eaac26263dd.png)

**Comment by fandongminghitmt** [comment]
thank you, so kind of you !

**Comment by pgii** [comment]
It's my pleasure.)))
![5e0b93facf](https://user-images.githubusercontent.com/4399076/96704068-0ccd8e80-13ad-11eb-8361-0c0b4b2570d0.png)
